parallel-install/pkg/components/prerequisites.go

```diff
@@ -29,7 +31,7 @@ func NewPrerequisitesProvider(overridesProvider overrides.OverridesProvider, res
 		HelmTimeoutSeconds:            cfg.HelmTimeoutSeconds,
 		BackoffInitialIntervalSeconds: cfg.BackoffInitialIntervalSeconds,
 		BackoffMaxElapsedTimeSeconds:  cfg.BackoffMaxElapsedTimeSeconds,
-		Log:                           cfg.Log,
+		Log:                           logger.NewLogger(cfg.Verbose),
```

Comment:
I don't think lower-level packages should actually care about logging level. Logger should be configured somewhere higher in the stack, and the components should be given ready-to-use logger. What's the value of propagating information about "Verbosity" across all layers?

Reply:
Verbosity flag was required to instantiate separate logger for every component. That's the way it was discussed to be done. After followup, we have decided to change the strategy.